examples/vpc-with-multiple-az/monitoring/main.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_cloudwatch_dashboard" "main" {
```

Comment:
The build fails because this file is not properly formatted according to `terraform fmt -check`